API-strategie-extensies/ext-geospatial.md

```diff
@@ -196,16 +205,14 @@ The API should be able to handle the following scenarios based on the rules stat
 
 Use the following URIs to specify the CRS:
 
-|CRS|URI|
-|-|-|
-|ETRS89, 2D, European|http://www.opengis.net/def/crs/EPSG/9.9.1/4258|
-|ETRS89, 3D, European|http://www.opengis.net/def/crs/EPSG/9.9.1/4937|
-|WGS84, global|http://www.opengis.net/def/crs/OGC/1.3/CRS84|
-|Web Mercator, global|http://www.opengis.net/def/crs/EPSG/9.9.1/3857|
-|RD/Amersfoort, 2D, Dutch|http://www.opengis.net/def/crs/EPSG/9.9.1/28992|
-|RD/Amersfoort + NAP, 3D, Dutch|http://www.opengis.net/def/crs/EPSG/9.9.1/7415|
-
-<!-- Deze URIs moeten gecheckt worden -->
+|Name|Dimension|Scope|URI|
+|-|-|-|-|
+|ETRS89 | 2D | European | http://www.opengis.net/def/crs/EPSG/9.9.1/4258|
+|ETRS89 | 3D | European | http://www.opengis.net/def/crs/EPSG/9.9.1/4937|
+|WGS 84 longitude-latitude | 2D | Global | http://www.opengis.net/def/crs/OGC/1.3/CRS84|
+|WGS 84 / Pseudo-Mercator | 2D | Global | http://www.opengis.net/def/crs/EPSG/9.9.1/3857|
```

Comment:
Hier staat Pseudo-Mercator, wat correct is, terwijl op regel 163 Web Mercator staat. Zou het voor de consistentie goed zijn om op regel 163 ook Pseudo-Mercator te gebruiken?